Library/Formula/owfs.rb

```diff
@@ -2,28 +2,16 @@
 
 class Owfs < Formula
   homepage 'http://owfs.org/'
-  url 'https://downloads.sourceforge.net/project/owfs/owfs/2.9p1/owfs-2.9p1.tar.gz'
-  version '2.9p1'
-  sha1 'd431ae5b8eec6988f8ee1e401325eba845cf3696'
+  url 'http://sourceforge.net/projects/owfs/files/owfs/3.1p0/owfs-3.1p0.tar.gz'
```

Comment:
Please keep an URL starting with `https://downloads.sourceforge.net` to get geolocation.